framework_tvb/tvb/interfaces/web/static/js/keycloak.js

```diff
@@ -0,0 +1,1686 @@
+/*
+ * Copyright 2016 Red Hat, Inc. and/or its affiliates
```

Comment:
We should add this a a 3rd party dependency in TVB_distribution and copy its original license next to the others

Reply:
Adding the original license file and one entry in packages_accepted.xml is generally enough for our dependencies.
But for the JS libraries, we do not have the "smart" auto detect feature, as we currently do with Python dependencies. So, you need to explicitly list keycloack JS into the EXTRA_MODULES dict (line 100 in tvb_build.third_party_licenses.package_finder.py